lib/src/components/NxNexusPageHeader/NxNexusPageHeader.scss

```diff
@@ -0,0 +1,138 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-container-helpers';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-page-header--nexus {
+  .nx-page-header__inner {
+    padding-left: 0;
+  }
+
+  .nx-product {
+    display: -ms-grid;
+    display: grid;
+    -ms-grid-columns: 72px 16px 1fr;
+    grid-template-columns: 72px 1fr;
+    -ms-grid-rows: 39px 0 auto;
+    grid-template-rows: 39px auto;
+    gap: 0 16px;
+    grid-template-areas:
+      "logo top-row"
+      "logo bottom-row";
+
+    height: 72px;
+    margin-bottom: 0;
+  }
+
+  .nx-product__top-row {
```

Comment:
This element is too tall for its grid cell

Reply:
> This element is too tall for its grid cell

Cleaned up as part of: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/fb3ae07c988230b25e10d6b58146a8e517236ec0

Reply:
> Cleaned up as part of: fb3ae07

It's still too tall. 
![image](https://user-images.githubusercontent.com/3630091/98014484-b52a1c80-1dc9-11eb-9032-29bade367b29.png)

Reply:
> > Cleaned up as part of: [fb3ae07](https://github.com/sonatype/sonatype-react-shared-components/commit/fb3ae07c988230b25e10d6b58146a8e517236ec0)
> 
> It's still too tall.
> ![image](https://user-images.githubusercontent.com/3630091/98014484-b52a1c80-1dc9-11eb-9032-29bade367b29.png)

Better layout: https://github.com/sonatype/sonatype-react-shared-components/pull/189/commits/336c0071698f0702fc770aee6bdc93070e4e2849

Reply:
> Better layout: 336c007

Much better.  A minor note though, the version and name are inline-block while the brand and byline are inline.  The inline-block elements have a slightly different height than their inline counterparts.  It doesn't affect the layout though, so up to you if you want to mess with it.